.golangci.yaml

```diff
@@ -0,0 +1,51 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  maligned:
+    suggest-new: true
+  goimports:
+    local-prefixes: github.com/Schrodinger-Hat/Daje
+  dupl:
+    threshold: 100
+  goconst:
+    min-len: 2
+    min-occurrences: 2
+linters:
+  disable-all: true
```

Comment:
Is it a best practice to disable all linters and then enable them all?

Reply:
I have enabled only a few of them. Not all the linters inside golangci are useful, I have found them as a good entry point for a project :smile: